### DISCUSSION TITLE
Duda sobre enunciado en Colaborador

### DISCUSSION THREAD

**Discussion by Luca-Aiello** [dds-jv/foro#418] [TPA - 2024] [answered]
Buen día profe, tenemos la siguiente duda con respecto a la esta parte del enunciado: 

Colaborador:
**Además, la persona deberá detallar de qué forma puede colaborar**

Esto quiere decir que debe haber una variable, o lista, en la que se registre como va a contribuir y luego que realice la o las contribuciones, o directamente intenta realizar una o mas alternativas de contribución.

**Comment by flbulgarelli** [comment, marked as answer]
:wave: ¡Hola!

No estoy seguro de haber entendido la idea de "variable o lista". Lo que se espera es que quien contribuya pueda elegir, al momento de realizar su contribución, la manera en que lo hará. Esto no impide que en el futuro haga contribuciones de otro tipo, ni exige que la persona tenga una "forma de contribuir" predeterminada: el tipo de contribución es algo específico de cada contribución que realice. 

¡Saludos!

**Comment by Luca-Aiello** [comment]
Buenas tardes profe!
Gracias por la respuesta, ya quedo claro.
Saludos!
